iceoryx_posh/test/integrationtests/service_discovery/test_roudi_findservice.cpp

```diff
@@ -40,6 +40,70 @@ TEST_F(RoudiFindService_test, OfferSingleMethodServiceSingleInstance)
     ASSERT_THAT(*instanceContainer.begin(), Eq(IdString("instance1")));
 }
 
+TEST_F(RoudiFindService_test, OfferServiceWithDefaultDescription)
+{
+    senderRuntime->offerService(iox::capro::ServiceDescription());
+    this->InterOpWait();
+    iox::runtime::InstanceContainer instanceContainer;
+    receiverRuntime->findService(iox::capro::ServiceDescription(), instanceContainer);
+
+    ASSERT_THAT(instanceContainer.size(), Eq(1u));
+    senderRuntime->stopOfferService(iox::capro::ServiceDescription());
+}
+
+TEST_F(RoudiFindService_test, ReOfferSingleMethodServiceSingleInstance)
+{
+    senderRuntime->offerService({"service1", "instance1"});
+    this->InterOpWait();
+    iox::runtime::InstanceContainer instanceContainer;
+    receiverRuntime->findService({"service1", "instance1"}, instanceContainer);
+
+    ASSERT_THAT(instanceContainer.size(), Eq(1u));
+    ASSERT_THAT(*instanceContainer.begin(), Eq(IdString("instance1")));
+
+    senderRuntime->stopOfferService({"service1", "instance1"});
+    this->InterOpWait();
+    instanceContainer.clear();
+    receiverRuntime->findService({"service1", "instance1"}, instanceContainer);
+
+    ASSERT_THAT(instanceContainer.size(), Eq(0u));
```

Comment:
Are you testing that stop offer works here? If so please create a separate test for it!
Please try to avoid to test multiple things in one tests.

Reply:
no, i am checking for a sequence, start -> stop -> start

Reply:
Could you please then add a stopOffer is working test.
And then you could remove this `ASSERT_THAT` since you verified it in the stopOffer test

Reply:
yeah,  test for stopOffer works is present at https://github.com/eclipse/iceoryx/blob/master/iceoryx_posh/test/integrationtests/service_discovery/test_roudi_findservice.cpp line148